index.html

```diff
@@ -5,184 +5,285 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
 </head>
 
 <body>
+    <header>
+        <a id="logo" href="#home">Temesgen</a>
+
+
+        <nav>
+
+            <ul class="nav-ul">
+                <li>
+                    <a href="#portfolio">Portfolio</a></li>
+                <li>
+                    <a href="#about">About</a></li>
+                <li>
+                    <a href="#contact">Contact</a></li>
+                <li><img class="nav-pic" src="./img/icon-menu.png" alt=""></li>
+            </ul>
+
+        </nav>
+    </header>
+    <section id="home" class="home">
+        <div class="home-content">
+            <h1>
+                I'm Temesgen <br> Glad to see you!
+            </h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+
+            <p class="connect">LET'S CONNECT</p>
+            <ul>
+                <li>
+                    <a href="twitter.com"><img src="./img/Vector.png" alt=""></a>
+                </li>
+                <li>
+                    <a href="medium.com"><img src="./img/Vector2.png" alt=""></a>
+                </li>
+                <li>
+                    <a href="linkedin.com"><img src="./img/Linkedin-icon.png" alt=""></a>
+                </li>
+                <li>
+                    <a href="github.com"><img src="./img/Vector3.png" alt=""></a>
+                </li>
+                <li>
+                    <a href="angellist.com"><img src="./img/Vector4.png" alt=""></a>
+                </li>
+            </ul>
+
+        </div>
```

Comment:
- [ ] I think it would be better if we improve the styling of the **Heading** section to make stick to the one from Figma by using the correct `color` on the **text** and using correct **spacing** at the left, then your project will stick to the Figma design. **Make sure you apply the use of the correct text color on all text of your page**.

- [ ] **Make sure you use the correct spacing(margin or padding)** at the **left** and **right** for the whole page to make your project stick to the Figma design. I suggest you check from the **Header** to the **Footer** and use the correct spacing. I know you can do it :muscle: :muscle: